projects/sonar/src/styles.scss

```diff
@@ -62,5 +62,23 @@ pre {
 }
 
 .card.wrapper {
-  @extend .my-3
+  border-radius: 0;
+  border: none;
+  @extend .py-1;
+
+  .card-header {
+    border: none;
+    background: none;
+    padding-left: 0;
+  }
+
+  .card-body {
+    padding-left: 0;
+  }
+
+  .border-left {
+    padding-left: 1rem !important;
+    margin-bottom: 1rem !important;
+    border-left: 3px solid #dee2e6 !important
```

Comment:
is this color could be defined into `variables.scss` file.

Reply:
I promise, if I use this color one more time, I will set a variable for it!